### PR TITLE
feature/Configure Withdraw page DAO-587

### PR DESCRIPTION
**Pull request by RakeshUP**

### CODE REVIEW

File: packages/web-app/src/pages/newWithdraw.tsx [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+  to: Address;
+  tokenSymbol: string;
+  tokenAddress: Address;
+};
+
+const steps = {
+  configure: 1,
+  setupVoting: 2,
+  defineProposal: 3,
+  reiewProposal: 4,
```
Comment (WissenIstNacht) [inline packages/web-app/src/pages/newWithdraw.tsx]: typo :p

**Review by sepehr2github — APPROVED** [review]
image
Thank you Rakash! btw Please enable the back button! thank you.

File: packages/web-app/src/pages/newWithdraw.tsx [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+  const {t} = useTranslation();
+  const {open} = useWalletMenuContext();
+  const {currentStep, prev, next} = useStepper(TOTAL_STEPS);
+  const {control} = useForm<FormData>({defaultValues});
+  const {connect, isConnected, account, ensName, ensAvatarUrl}: useWalletProps =
+    useWallet();
+
+  /** Toggle wallet */
+  const handleWalletButtonClick = useCallback(() => {
+    console.log('trigger');
```
Comment (WissenIstNacht) [inline packages/web-app/src/pages/newWithdraw.tsx]: Still needed?

File: packages/web-app/src/pages/newWithdraw.tsx [hunk 3]
```diff
@@ -0,0 +1,161 @@
...
+    isConnected() ? open() : connect('injected');
+  }, [connect, isConnected, open]);
+
+  return (
+    <>
+      <NavigationBar>
+        <HStack>
+          <InsetButton>
```
Comment (WissenIstNacht) [inline packages/web-app/src/pages/newWithdraw.tsx:68]: Not sure if you forgot or if it's to be done in another PR, but the button isn't connected to any logic.

File: packages/web-app/src/pages/newWithdraw.tsx [hunk 4]
```diff
@@ -0,0 +1,161 @@
...
+            <InsetIconContainer>
+              <IconChevronLeft />
+            </InsetIconContainer>
+            <InsetButtonText>{t('allTransfer.newTransfer')}</InsetButtonText>
+          </InsetButton>
+
+          <ButtonIcon
```
Comment (WissenIstNacht) [inline packages/web-app/src/pages/newWithdraw.tsx:76]: (Same here)

File: packages/web-app/src/locales/en/common.json [hunk 5]
```diff
@@ -67,5 +72,15 @@
...
     "amountSubtitle": "Define amount to deposit",
     "referenceSubtitle": "Add a reference copy to identify this transaction later on"
+  },
+  "newWithdraw": {
+    "withdrawAssets": "Withdraw Assets",
+    "configureWithdraw": {
+      "title": "Configure Withdraw",
+      "subtitle": "Enter the desired token and its number for transmission. Furthermore, a reception address is necessary.",
+      "toSubtitle": "Define receiver of tokens",
+      "tokenSubtitle": "Select token you’d like to withdraw",
```
Comment (WissenIstNacht) [inline packages/web-app/src/locales/en/common.json]: Fancy apostrophe. Is this intentional?

File: packages/web-app/src/containers/configureWithdraw/index.tsx [hunk 6]
```diff
@@ -0,0 +1,72 @@
...
+    <>
+      <FormItem>
+        <Label
+          label={t('labels.to')}
+          helpText={t('newWithdraw.configureWithdraw.toSubtitle')}
+        />
+        <ValueInput
+          placeholder={t('placeHolders.walletOrEns')}
+          adornmentText={t('labels.paste')}
+          onAdornmentClick={() => navigator.clipboard.readText().then(alert)}
```
Comment (WissenIstNacht) [inline packages/web-app/src/containers/configureWithdraw/index.tsx:30]: Todo in the input validation PR?

**Review by WissenIstNacht — APPROVED** [review]
Hey, thanks! Left a couple of minor comments. Also, there's a few functionalities missing. Wasn't sure if you left them for other PRs.

File: packages/web-app/src/locales/en/common.json (hunk 5 above)
Comment (RakeshUP) [reply packages/web-app/src/locales/en/common.json]: Copied the text from figma!

File: packages/web-app/src/pages/newWithdraw.tsx (hunk 4 above)
Comment (RakeshUP) [reply packages/web-app/src/pages/newWithdraw.tsx:76]: Didn't know the purpose of this button. Added a TODO

File: packages/web-app/src/pages/newWithdraw.tsx (hunk 3 above)
Comment (RakeshUP) [reply packages/web-app/src/pages/newWithdraw.tsx:68]: The back button will work from the recent change

File: packages/web-app/src/containers/configureWithdraw/index.tsx (hunk 6 above)
Comment (RakeshUP) [reply packages/web-app/src/containers/configureWithdraw/index.tsx:30]: Yes
